navigation/AppNavigation.js

```diff
@@ -1,86 +1,19 @@
-import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
-import HomeScreen from "../Screens/HomeScreen";
-import SignIn from "../Screens/SignIn";
-import SignUp from "../Screens/SignUp";
-import ParkedCarForm from "../Screens/ParkedCarForm";
-import Ionicons from "react-native-vector-icons/Ionicons";
-import { Image, StyleSheet, Text, TouchableOpacity, View } from "react-native";
-import MaterialCommunityIcons from "react-native-vector-icons/MaterialCommunityIcons";
-import { NavigationContainer } from "@react-navigation/native";
-import { createStackNavigator } from "@react-navigation/stack";
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import SignIn from '../Screens/SignIn';
+import SignUp from '../Screens/SignUp';
+import Tabs from './tabs';
+
 const Stack = createStackNavigator();
-const Tab = createBottomTabNavigator();
 
-const Tabs = () => {
+const AppNavigation = () => {
   return (
-    <Tab.Navigator
-      screenOptions={{
-        headerShown: false,
-        tabBarActiveTintColor: "#2e64e5",
-        tabBarStyle: [
-          {
-            display: "flex",
-          },
-          null,
-        ],
-      }}
-    >
-      <Tab.Screen
-        name="Home"
-        component={HomeScreen}
-        options={({ route }) => ({
-          tabBarLabel: "Home",
-          //   tabBarVisible: route.state && route.state.index === 0,
-          tabBarIcon: ({ color, size }) => (
-            <MaterialCommunityIcons
-              name="home-outline"
-              color={color}
-              size={size}
-            />
-          ),
-        })}
-      />
-      <Tab.Screen
-        name="Camera"
-        component={ParkedCarForm}
-        options={({ route }) => ({
-          //   tabBarVisible: getTabBarVisibility(route),
-          // Or Hide tabbar when push!
-          // https://github.com/react-navigation/react-navigation/issues/7677
-          // tabBarVisible: route.state && route.state.index === 0,
-          //   tabBarLabel: "Home",
-          tabBarIcon: ({ color, size }) => (
-            <Ionicons name="camera-outline" color={color} size={size} />
-          ),
-        })}
-      />
-      <Tab.Screen
-        name="Profile"
-        component={SignIn}
-        options={{
-          // tabBarLabel: 'Home',
-          tabBarIcon: ({ color, size }) => (
-            <Ionicons name="person-outline" color={color} size={size} />
-          ),
-        }}
-      />
-    </Tab.Navigator>
+    <Stack.Navigator initialRouteName="HomeScreen" screenOptions={{ headerShown: false }}>
+      <Stack.Screen name="SignIn" component={SignIn} />
+      <Stack.Screen name="SignUp" component={SignUp} />
```

Comment:
These screens still have the nav bar. I can look at the other screens without signing in?

Reply:
Don't try to merge this branch, still on-going issues